app/src/lib/parser/alternate_lookups.js

```diff
@@ -0,0 +1,72 @@
+import {TOKEN_TYPE, create_token} from "./token";
```

Comment:
```suggestion
import {TOKEN_TYPE, create_token} from './token'
```

Reply:
take a look at the `format` scripts in the `package.json` to take care of little stuff like this before committing.